tests/apollo/Apollo.test.ts

```diff
@@ -289,38 +289,178 @@ describe("Apollo", () => {
     expect(privateKey.isDerivable()).to.be.equal(true);
   });
 
-  it("Should derive secp256k1 privateKey the same way as if we create a new key in Apollo.", async () => {
-    const seed = apollo.createRandomSeed().seed;
-    const privateKey = apollo.createPrivateKey({
+  // without seed being added to the key, secp.derive can't create the same derivation as Apollo
+  it.skip("Should derive secp256k1 privateKey the same way as if we create a new key in Apollo.", async () => {
```

Comment:
this test is now invalid as without the seed Secp.derive can't get the same results, unless we change how Apollo.createPrivateKey works to return the master key (without deriving 'm/0/0/0')